keyboards/keycapsss/o4l_5x12/info.json

```diff
@@ -1,7 +1,13 @@
 {
-    "keyboard_name": "O4L 5x12",
+    "keyboard_name": "O4L:5x12",
```

Comment:
Is this colon supposed to be here?

Reply:
Yes.
![image](https://user-images.githubusercontent.com/4781841/182008689-f7782ccd-3ad0-4743-afdc-b524204a238f.png)

Reply:
Ah, I misread the `config.h` changes.